### DISCUSSION TITLE
Can we get the old website back?

### DISCUSSION THREAD

**Discussion by aykaramba** [nesbox/TIC-80#1928] [General]
Hi!

Is there any chance to roll the website back to the old theme or are there plans to port the old theme to the new site?

Tic80 is such a cool piece of software, the retro vibe of the site was perfect and I hope others might agree as well  ...   although maybe chime in if you disagree.

**Comment by joshgoebel** [comment]
Yikes, I have to agree, what happened to our cool website?

**Comment by nesbox** [comment]
I'm thinking of opening website sources where we could improve it together.
As for the new theme, I disabled the old one because there were problems with the `/learn` page, I hope to fix it soon.

**Reply by aykaramba** [reply]
That would be cool.  Just to note, my question was just out of curiosity because I liked the old theme so much.  Sure, we can dig in and see what can be done.

**Comment by bettywolfy** [comment]
Oh man, you didn't.
